src/components/AppBar.tsx

```diff
@@ -147,77 +149,84 @@ function AppBar(): JSX.Element {
+                                    src={theme === 'dark' ? LogoDark : Logo}
+                                    alt="GoodDollar"
+                                    className="hidden w-auto site-logo lg:block"
+                                />
+                                <img src={LogoMobile} alt="GoodDollar" className="w-auto h-7 lg:hidden" />
+                            </LogoWrapper>
+                        </div>
+
+                        <div className="flex flex-row space-x-2">
+                            <div className="flex flex-row items-center space-x-2">
+                                <div className="ml-10 text-sm whitespace-nowrap lg:text-base">
+                                    {G$Price ? `1,000G$ = ${G$Price.multiply(1000).toFixed(3)}USD` : ''}
+                                </div>
+                                {chainId && <Web3Faucet />}
+                                <button
+                                    onClick={() => {
+                                        setSidebarOpen(!sidebarOpen)
+                                    }}
+                                    className="inline-flex items-center justify-center p-2 rounded-md mobile-menu-button focus:outline-none"
+                                >
+                                    <span className="sr-only">{i18n._(t`Open main menu`)}</span>
+                                    {sidebarOpen ? (
+                                        <X title="Close" className="block w-6 h-6" aria-hidden="true" />
+                                    ) : (
+                                        <Burger title="Burger" className="block w-6 h-6" aria-hidden="true" />
+                                    )}
+                                </button>
+                            </div>
+                            <div className="fixed bottom-0 left-0 flex flex-row items-center justify-center w-full p-4 lg:w-auto lg:relative lg:p-0 actions-wrapper ">
+                                <div className="flex items-center justify-center w-full space-x-2 sm:justify-center">
+                                    <div className="hidden xs:inline-block">
                                         <Web3Network />
-                                      </div>
-                                      {account && chainId && userEthBalance ? (
-                                          <DivOutlined className="pr-1">
+                                    </div>
+                                    {account && chainId && nativeBalance ? (
+                                        <DivOutlined className="pr-1">
                                             <div className="w-auto flex items-center rounded p-0.5 whitespace-nowrap   cursor-pointer select-none pointer-events-auto">
                                                 <div className="px-3 py-2 bold">
-                                                    {userEthBalance?.toSignificant(4)}{' '}
```

Comment:
the UI should be the one deciding how to display significant digits

Reply:
Formatting is done as before now